sycl/test/check_device_code/vector/vector_math_ops.cpp

```diff
@@ -1,7 +1,8 @@
 // NOTE: Assertions have been autogenerated by utils/update_cc_test_checks.py UTC_ARGS: --version 4
 // NOTE: ..., followed by some manual cleanup.
 
-// RUN: %clangxx -I %sycl_include -fno-discard-value-names -S -emit-llvm -fno-sycl-instrument-device-code -Xclang -disable-lifetime-markers -fsycl-device-only %s -o - | FileCheck %s
+// RUN: %clangxx -I %sycl_include -S -emit-llvm -fno-discard-value-names -fno-sycl-instrument-device-code -Xclang -disable-lifetime-markers -fsycl-device-only %s -o - | FileCheck %s
+// RUN: %if preview-breaking-changes-supported %{ %clangxx -I %sycl_include -S -emit-llvm -fno-discard-value-names -fpreview-breaking-changes -fno-sycl-instrument-device-code -Xclang -disable-lifetime-markers -fsycl-device-only %s -o - | FileCheck %s -check-prefix=CHECK-ARR-STORAGE %}
```

Comment:
I'm only interested in diff here. Can we drop the first RUN line and use default CHECK prefix for the second?